app/src/main/kotlin/com/wire/android/ui/theme/WireColorScheme.kt

```diff
@@ -0,0 +1,126 @@
+package com.wire.android.ui.theme
+
+import androidx.compose.material3.ColorScheme
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.graphics.Color
+import io.github.esentsov.PackagePrivate
+
+@Immutable
```

Comment:
What's this annotation aimed for?

Reply:
> using @Immutable is a promise that the value will never change
most samples, even in the official documentation, have such annotation, it's not particularly useful, but it doesn't do any harm either 🙃